### PR TITLE
OJ-2453: Add `evidence_requested` to session from JAR request

### PR DESCRIPTION
**Pull request by surajkumar**
### What and why did it change
Added `evidence_requested` details into the session table because ipv-core now sends `evidence_requested` within the signed JAR payload. This has superseded the `scope` value that was provided however this was not used in common-lambda.

### Issue tracking
- [OJ-2453](https://govukverify.atlassian.net/browse/OJ-2453)

[OJ-2453]: https://govukverify.atlassian.net/browse/OJ-2453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by liamStockwellGDS — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,39 +57,24 @@ If you have not installed `pre-commit` then please do so [here](https://pre-comm
 
 ## Run Cucumber tests
 
-Below runs by using the AWS stub, with the following defaults:
+Below runs and uses the core stub as default
 
-- DEFAULT_CLIENT_ID="ipv-core-stub-aws-build"
-- ENVIRONMENT=DEV
-- CRI_DEV=common-lambda-dev
+`STACK_NAME=di-ipv-cri-common-api-your-stack-name ENVIRONMENT=dev API_GATEWAY_ID_PRIVATE=xxxx IPV_CORE_STUB_BASIC_AUTH_USER=xxxx IPV_CORE_STUB_BASIC_AUTH_PASSWORD=xxxx IPV_CORE_STUB_URL="https://cri.core.stubs.account.gov.uk" gradle integration-tests:cucumber
```
Comment (jkunle) [inline README.md:62]: Looks like you are about to get rid of vital information, I suppose maybe it shouldn't live here. we should discuss maybe the wording and usefulness. My gut is you have found this useful and want to stream the content

Maybe remove changes to the readme for now, or add to it but I think we should remove from it at least not until may we run through it together

File: lambdas/tests/unit/services/session-request-validator.test.ts [hunk 2]
```diff
@@ -182,11 +183,48 @@ describe("session-request-validator.ts", () => {
...
+                    redirect_uri: redirect_uri,
+                    state: state,
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () => sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id)).resolves;
+        });
+
         xit("should fail to validate the jwt if scope is missing", async () => {
```
Comment (jkunle) [inline lambdas/tests/unit/services/session-request-validator.test.ts:228]: We should remove the commented test if it is about scope

Comment (surajkumar) [reply lambdas/tests/unit/services/session-request-validator.test.ts]: I have removed all references of scope 👍

File: README.md (hunk 1 above)
Comment (surajkumar) [reply README.md:62]: @liamStockwellGDS please can you look into this?

**Review by jkunle — APPROVED** [review]

**Pull request merged**